tests/conftest.py

```diff
@@ -430,3 +462,157 @@ def plates_lookup_with_samples(samples, priority_samples):
 @pytest.fixture
 def plates_lookup_without_samples(samples, priority_samples):
     return PLATES_LOOKUP_WITHOUT_SAMPLES
+
+
+@pytest.fixture
+def mocked_rabbit_channel(app):
+    with app.app_context():
+        mocked_broker = MagicMock()
+        with patch("lighthouse.classes.services.warehouse.Broker", return_value=mocked_broker):
+            mocked_channel = MagicMock()
+            mocked_broker.__enter__.return_value = mocked_channel
+
+            yield mocked_channel
+
+
+@pytest.fixture
+def cherrytrack_mock_run_info(
+    app, mocked_responses, run_id, cherrytrack_run_info_response, cherrytrack_mock_run_info_status
+):
+    run_url = f"{app.config['CHERRYTRACK_URL']}/automation-system-runs/{run_id}"
+
+    mocked_responses.add(
+        responses.GET,
+        run_url,
+        json=cherrytrack_run_info_response,
+        status=cherrytrack_mock_run_info_status,
+    )
+    yield
+
+
+@pytest.fixture
+def cherrytrack_mock_source_plates_status():
```

Comment:
These returned HTTPStatus don't feel right.

Reply:
Yes, the testing fixtures dont feel right, I think that will need a good rethinking/refactor but I'm not having better ideas for the moment.

Reply:
This was defined as fixture to be able to redefine it in the tests if it is needed.